### PR TITLE
ricardian contract format

### LINKED ISSUE #294: Ricardian Contract format.
The ricardian contract is used to describe the condition and action of PF.
It will be signed and recorded in the blockchain by the publisher.
When PF voting procedure is executed, each voting transaction must refer to the ricardian contract
and signed voting transaction will be the evidence that voter has agreed on the contents of the ricardian contract.
It will need to first define the ricardian contract format.

### PR DESCRIPTION
**Pull request by soonkuk**
### Github Issue

resolves #294

### Background
Ricardian contract is human readable contract which represent PF operation.
Proposer should make ricardian contract of the PF.

### Solution
This PR define the field of the ricardian contract.
Ricardian contract can be converted to markdown format.
The field is chosen from the document which describe the PF_R_00. But there is a possibility to change the some fields.

### CODE REVIEW

**Comment by kfangw** [comment]
Please attach the issue number.

**Comment by kfangw** [comment]
@soonkuk Could you explain the flow or usecase of this feature?

**Comment by soonkuk** [comment]
This is the flow of feature I thought.
A PF proposal is represented by means of ricardian contract.
When the proposer submits a proposal, he sends a transaction which contain the data according to fields of ricardian struct about PF.
In blockchain, an instance of ricardian contract struct type is generated and stored.
If client wants human readable format of proposal, client requests to the blockchain and receive a converted markdown format.

**Comment by kfangw** [comment]
> This is the flow of feature I thought.
> A PF proposal is represented by means of ricardian contract.
> When the proposer submits a proposal, he sends a transaction which contain the data according to fields of ricardian struct about PF.
> In blockchain, an instance of ricardian contract struct type is generated and stored.
> If client wants human readable format of proposal, client requests to the blockchain and receive a converted markdown format.

So, how can I submit the proposal?
Is there a API for it?

File: lib/common/ricardian/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# How to make a ricardian contract for PF proposal.
```
Comment (kfangw) [inline lib/common/ricardian/README.md]: Could you move this file to docs/?

Comment (soonkuk) [reply lib/common/ricardian/README.md]: Yes, I moved it.

**Comment by spikeekips** [comment]
> @kfangw @spikeekips If parsing needed then, where to put the parsed data?

I expected the parsed data can be printed in client-side.

**Comment by kfangw** [comment]
> > @kfangw @spikeekips If parsing needed then, where to put the parsed data?
> 
> I expected the parsed data can be printed in client-side.

To support this requirement, @soonkuk you need to make a client code which parses the encoded text and print it.

**Comment by soonkuk** [comment]
What is the purpose of parsing ricardian contract in client side?

**Comment by spikeekips** [comment]
@soonkuk client will show the details of PR and PR is markdown, so OR can be parsed.

**Comment by soonkuk** [comment]
I am discussing with PF team about contents in ricardian contract.
They are being reviewed by a external agency about ricardian contract contents.
Maybe next week it'll be finished.

**Comment by spikeekips** [comment]
@soonkuk Please update the current status.

**Pull request merged**